galasa-ui/src/app/cookies/route.ts

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
Not sure about making an API route just for getting and deleting cookies - maybe server actions are more suitable here?